### PR TITLE
fix docstrings of xlogx and xlogy

### PR DESCRIPTION
**Pull request by cossio**
cc @nalimilan

### CODE REVIEW

File: src/basicfuns.jl [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 # scalar functions
 """
-    xlogx(x::Real)
+    xlogx(x)
```
Comment (nalimilan) [inline src/basicfuns.jl:5]:
```suggestion
    xlogx(x::Number)
```

File: src/basicfuns.jl [hunk 2]
```diff
@@ -17,9 +17,9 @@ function xlogx(x::Number)
 end
 
 """
-    xlogy(x::Real, y::Real)
+    xlogy(x, y)
```
Comment (nalimilan) [inline src/basicfuns.jl:20]:
```suggestion
    xlogy(x::Number, y::Number)
```

Comment (cossio) [reply src/basicfuns.jl]: Thanks!

File: src/basicfuns.jl (hunk 1 above)
Comment (cossio) [reply src/basicfuns.jl]: Thanks!

**Comment by cossio** [comment]
`README.md` was out of date so I updated it here.

**Comment by nalimilan** [comment]
Thanks!

**Pull request merged**